mlflow/transformers.py

```diff
@@ -1647,8 +1650,8 @@ def trim_input(data_in, data_out):
                     data_out = data_out[len(data_in) :].lstrip()
                     if data_out.startswith("A:"):
                         data_out = data_out[2:].lstrip()
-                for to_replace, replace in replacements.items():
-                    data_out = data_out.replace(to_replace, replace)
+                    for to_replace, replace in replacements.items():
+                        data_out = data_out.replace(to_replace, replace)
```

Comment:
What if a user doesn't want the prompt, but wants to preserve new characeters in the output?

Reply:
I can add an additional kwargs entry for that